opm/autodiff/WellDensitySegmented.cpp

```diff
@@ -144,6 +145,122 @@ Opm::WellDensitySegmented::computeConnectionDensities(const Wells& wells,
 }
 
 
+std::vector<double>
```

Comment:
I can make WellState a template parameter. 
And add a trivial solvent_fraction to WellStateFullyImplicitBlackoil to avoid this copying.